examples/lessons/06/app.js

```diff
@@ -44,6 +44,7 @@ var filters = ['nearest', 'linear', 'mipmap'];
 const animationLoop = new AnimationLoop({
   // .context(() => createGLContext({canvas: 'lesson05-canvas'}))
   onInitialize: ({canvas, gl}) => {
+    addControls();
 
     addEvents(canvas, {
```

Comment:
We need this here, lesson 6 is not showing link to the WebGL example, and this fixes it.